packages/cli/src/bin/pipcook.ts

```diff
@@ -3,12 +3,19 @@
 import * as semver from 'semver';
 import * as chalk from 'chalk';
 import * as program from 'commander';
+import * as fs from 'fs-extra';
 import { join } from 'path';
 import { constants } from '@pipcook/core';
 import { readJson, mkdirp, remove } from 'fs-extra';
 import { StandaloneRuntime } from '../runtime';
-import { logger, dateToString } from '../utils';
+import { logger, dateToString, execAsync } from '../utils';
+const download = require('download-git-repo');
```

Comment:
```suggestion
const download = import('download-git-repo');
```

Reply:
`download-git-repo` has not types, so if `import` it, we will get error `ts(7016)`.

Reply:
We need to leave a comment for this.